docs/reference/scripting/access-fields.asciidoc

```diff
@@ -84,4 +84,42 @@ operating on them:
 ZonedDateTime start = field('start').get(null);
 ZonedDateTime end = field('end').get(null);
 return start == null || end == null ? -1 : ChronoUnit.MILLIS.between(start, end)
-----
\ No newline at end of file
+----
+
+[discrete]
+=== Supported mapped field types
+The following table indicates the mapped field types that the `field` API
+supports. For each supported type, values are listed that are returned by the
+`field` API (from the `get` and `as<Type>` methods) and the `doc` map (from the
+`getValue` and `get` methods). 
+
+NOTE: Some fields are not currently supported. For the most current list of
+supported fields, refer to {es-issue}79105[#79105].
+
+[cols="1,1,1",options="header",]
+|========
+|Mapped field type
+|Returned type from `field` (`get`,`asType`)
```

Comment:
asType becomes something like asBigInteger or asString which I'm not sure is clear from this. Is there a way to show that Type -> BigInteger, etc.